Apps/Contoso.WPF.Puppet/MainWindow.xaml.cs

```diff
@@ -116,20 +120,27 @@ private void CountryCodeEnabled_Checked(object sender, RoutedEventArgs e)
             }
             if (!CountryCodeEnableCheckbox.IsChecked.Value)
             {
-               CountryCodeText.Text = "";
-                AppCenter.SetCountryCode(null);
+                CountryCodeText.Text = "";
             }
             else
             {
-                CountryCodeText.Text = RegionInfo.CurrentRegion.TwoLetterISORegionName;
-                AppCenter.SetCountryCode(CountryCodeText.Text);
```

Comment:
We should also keep the set API call as we can also disable/enable analytics to avoid restart. It's also good for event/crash to have it real time. Both use cases are good.